saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -161,12 +161,19 @@ class BulkAttributeValueInput(InputObjectType):
     id = graphene.ID(description="ID of the selected attribute.")
     values = graphene.List(
         graphene.String,
-        required=True,
+        required=False,
         description=(
             "The value or slug of an attribute to resolve. "
             "If the passed value is non-existent, it will be created."
         ),
     )
+    boolean = graphene.Boolean(
```

Comment:
Maybe we should add `validate_one_of_args_is_in_mutation` to mutation which use this input?

Reply:
Validation is handled via `AttributeAssignmentMixin` like in `ProductVariantCreate`